### ISSUE TITLE
When log.debug(err) and err is a JS error object only {} gets logged.

### ISSUE THREAD

**Issue by codeowl1** [darrylwest/simple-node-logger#18]
Hi there,

When I log an the JS error object from an exception handler only an empty object gets logged.

Here is a sample program to replicate the issue:

```
const log = require('simple-node-logger').createSimpleLogger({ level: 'all' });

try {
	var o = {};
	var i = o.test.test;
} catch (err) {
	log.debug(err);
}
```

Here is the output:

$ node test-simple-logger.js
17:38:33.555 DEBUG {}

However when I set a breakpoint (using MS VS) I can see that the error object is not an empty object, but has a message property that contains the error message.

I am running nodejs v4.5.0 on Windows 10 with simple-node-logger 0.93.24.

**Comment by codeowl1** [comment]
I have fixed this one.  In the file: \simple-node-logger\lib\AbstractAppender.js
Replace the function:  formatObject, with the following:
```javascript
    this.formatObject = function(value) {
        if (!value) {
            return '';
        }
        if (dash.isObject( value )) {
            if (value instanceof Error) {
                return (value.message ? value.message : '') + (value.stack ? value.stack : '')
            }
            try {
                return JSON.stringify( value );
            } catch (ignore) {
                return 'json error: ' + value.toString();
            }
        } else {
            var s = value.toString();
            if (s === '[object Object]') {
                return util.inspect( value );
            } else {
                return s;
            }
        }
    };
```

The key part I have added is to detect and log javascript error objects:
```javascript
if (value instanceof Error) {
    return (value.message ? value.message : '') + (value.stack ? '\n'+value.stack : '')
}
````

The above code logs both the message and the stack.  value.toString() only logs the message.
So the output is now:
```
 17:07:51.292 DEBUG Cannot read property 'test' of undefined
TypeError: Cannot read property 'test' of undefined
    at Object.<anonymous> (C:\test-simple-logger.js:27:16)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.runMain [as _onTimeout] (module.js:441:10)
    at Timer.listOnTimeout (timers.js:92:15)
```
Would be great to have this included in future releases :-)

**Comment by darrylwest** [comment]
will be fixed at version 0.93.29...
